server/core/src/dataObjOpr.cpp

```diff
@@ -315,6 +323,12 @@ getDataObjInfo(
         return UNMATCHED_KEY_OR_INDEX;
     }
 
+    if ( ( dataAccessTime =
+                getSqlResultByInx( genQueryOut, COL_D_ACCESS_TIME ) ) == nullptr ) {
+        log_api::info("{}: getSqlResultByInx for COL_D_ACCESS_TIME failed", __func__);
```

Comment:
may not be an error...

```suggestion
        log_api::error("{}: getSqlResultByInx for COL_D_ACCESS_TIME failed", __func__);
```

Reply:
done